Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/DriveBase.java

```diff
@@ -113,6 +131,28 @@ public void log() {
 		// SmartDashboard.putNumber("Right Distance", this.rightEncoder.get());
 	}
 
+	public double[] distanceTraveled() {
+		// TODO: This would be where the encoder logic would go to determine how far it
+		// has traveled.
+		// Things to consider 1. How will we reset distances between calls?
+		// 2. How will we ensure distances aren't reset until the previous command is
+		// done?
+		// 3. Will distance traveled only be allowed in autonomous?
+		// 4. How will we track distances between calls? I.e. pass it in as a parameter?
+		// Field on driveBase that gets reset?
+		
+		// TODO: See ReferenceDocuments/Talon SRX Victor SPX - Software Reference Manual.pdf
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)